cm-api-shim.js

```diff
@@ -18,7 +18,8 @@
 (function() {
   // Return early if CM API is not available or the new features are already
   // present.
-  if (!navigator.credentials || navigator.credentials.preventSilentAccess)
+  if (!window.PasswordCredential || !window.FederatedCredential || 
+      navigator.credentials.preventSilentAccess)
```

Comment:
Potentially we need to update line 55 as well, as it is theoretically possible that a browser implements the old CM API in conjunction with the WebAuthN API (this wouldn't be the case for Chrome though). What is your opinion here?

Reply:
I think it's possible that there will be a browser which implements `PasswordCredential` but `FederatedCredential`. I don't think we need to change this line.

Reply:
I think adding checks for `PasswordCredential` and `FederatedCredential` in line 69 and 78 would make sense.

Reply:
Done